.gitignore

```diff
@@ -13,6 +13,7 @@ docs/build/
 docs/site/
 docs/src/tutorials/
 docs/src/generated/
+docs/DUMMY_DIR/generated/
```

Comment:
Let's not call this `DUMMY_DIR`? Maybe `transient` or something?

Reply:
that's fine by me

Reply:
Changed to `transient_dir`